From 93c1fe9f941134d294f80a4bc619fb502193550e Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Thu, 12 Nov 2020 22:07:26 +0100
Subject: [PATCH] prom_init: enable verbose prints

References: bsc#1178142 bsc#1180759
Patch-mainline: never, debug patch - would need runtime enable

Enable verbose prints in ppc ofw kernel stub.

There has been issue with kernel requesting impossible configuration
from the firmware during feature negotioation. With this we should have
data in hand next time such error happens.

The downside is some extra noise on the console during early boot.

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 arch/powerpc/kernel/prom_init.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/powerpc/kernel/prom_init.c b/arch/powerpc/kernel/prom_init.c
index 923c2cc95d44..532fe870795a 100644
--- a/arch/powerpc/kernel/prom_init.c
+++ b/arch/powerpc/kernel/prom_init.c
@@ -9,7 +9,7 @@
  *    {engebret|bergner}@us.ibm.com 
  */
 
-#undef DEBUG_PROM
+#define DEBUG_PROM
 
 /* we cannot use FORTIFY as it brings in new symbols */
 #define __NO_FORTIFY
-- 
2.26.2

